Add `@show` and `@hide` directives (#2392)
nuwave/lighthouse: docs/master/api-reference/directives.md

}
```

## @hide

```graphql
"""
Hides field if environment is one of specified
"""
directive @hide(
    """

>     Specify which environments must not use this field, e.g. ["prod"].

Reviewer: ```suggestion
    Specify which environments exclude this element, e.g. `["prod"]`.
    Compared against the value returned from `app()->environment()`.
```

    """
    env: [String!]!
) repeatable on FIELD_DEFINITION
```

This directive lets you exclude some parts of schema in different environments. For example, you might 
remove debug information in `prod` environment:

```graphql
type Query {

>     debugInformation: String! @hide(env:["prod"])

Reviewer: ```suggestion
    debugInformation: String! @hide(env: ["prod"])
```

> 

Reviewer: ```suggestion
}
```

}
```

## @show

```graphql
"""
Includes the annotated element from the schema conditionally.
"""
directive @show(
    """

>     Specify which environments may use this field, e.g. ["testing"].

Reviewer: ```suggestion
    Specify which environments include this element, e.g. ["testing"].
```
